content/news/XXXX-XX-XX-mixxx-2-4-1-released.md

```diff
@@ -0,0 +1,148 @@
+title: Mixxx 2.4.1 released
+authors: Evelynne Veys
+tags: 2.4, 2.4.1, release announcement
+comments: yes
+status: draft
+
+####Dear Mixxx-ers,
+
+The Mixxx-developingteam started the year with good intentions and therefore imposed itself some deadlines. That is why only two months after version 2.4, 
+the next release: version 2.4.1 can be offered. In between versions with new features this version delivers some solutions for minor bugs and problems that 
+came with 2.4.
```

Comment:
Here is how I would tweak the first paragraph

```suggestion
The Mixxx developers started the year with good intentions and therefore imposed some deadlines on themselves.
That is why only two months after version 2.4, version 2.4.1 has been released.
This version delivers solutions for minor bugs and issues that came with 2.4.
```

Reply:
> Here is how I would tweak the first paragraph

done